iptables/iptables.go

```diff
@@ -512,8 +512,14 @@ func filterOutput(start time.Time, output []byte, args ...string) []byte {
 // Raw calls 'iptables' system command, passing supplied arguments.
 func (iptable IPTable) Raw(args ...string) ([]byte, error) {
 	if firewalldRunning {
+		// select correct IP version for firewalld
+		ipv := Iptables
+		if iptable.Version == IPv6 {
+			ipv = IP6Tables
+		}
+
 		startTime := time.Now()
-		output, err := Passthrough(Iptables, args...)
+		output, err := Passthrough(ipv, args...)
```

Comment:
why not pass `iptable.Version` directly ?

Reply:
because 
`iptable.Version = "IPV6"`
and 
`IP6Tables = "ipv6"`

Not sure if Firewalld likes uppercase.